user-guide/enterprise-auth/oauth2.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Configuring OAuth 2 Authentication
+description: Learn how to configure and connect to an enterprise identity provider using OAuth 2.
+position: 4
+publish: true
+slug: oauth2
+---
+
+# Configuring OAuth 2 Authentication
+
+NativeScript Sidekick gives you the ability to connect to enterprise authentication providers via OAuth 2. In this article we’ll look at the various data points NativeScript Sidekick needs to make the connection, and then look at a concrete example of a sample connection using Microsoft Azure.
+
+## Configuring your connection
+
+The first thing you need to do to configure an OAuth 2 connection is to visit the **Enterprise Auth** screen and select the **OAuth 2** radio button. You should see a form that looks like this.
+
+![](/images/enterprise-auth/oauth-form.png)
+
+Let’s start by looking at the fields you need to provide to connect to your enterprise authentication provider. Then, as an example, we'll explore how we'd complete the form for a project using Azure Active Directory.
+
+## Completing the connection setup
+
+Here is a list of the fields you need to provide in Sidekick, and a brief description of what they are.
+
+<table>
```

Comment:
I have changed the table from **markdown** to **HTML** one. On the markdown table, I cannot set properly a width for the columns and they look very bad. The first column is under 10% width and each word was on a separate row. With the HTML table it's still far from perfect, but it's still better. :)

The tables in the entire NS documentation need some love. I should probably raise this question again.